UKHomeOffice/cerberus-service
# COP-10118 - Fix check-in field not highlighting when there data change

## Description
This PR corrects a bug inadvertently introduced by myself for the highlighting of the **`Check-in`** field when the data has changed between versions.

## To Test
- Pull & run against dev.
- Find the following task e.g `AUTOTEST-11-02-2022-RORO-Accompanied-Freight-different-versions-task_757168:CMID=TEST`, this has 3 versions. In Version 2, the `Check-in` field and date should be highlighted.

## Developer Checklist

\* Required
